.rubocop.yml

```diff
@@ -1,6 +1,18 @@
 # Inherit style from another configuration
 inherit_from: .rubocop_todo.yml
 
+ClassLength:
+   Exclude:
+    - 'app/models/ability.rb'
+
+MethodLength: 
+ Exclude:
+    - 'app/models/ability.rb'
```

Comment:
Tried it on my machine and it seems that only the initialize method offends that cop. Please just disable that cop.